Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -68,10 +68,11 @@ public struct Conv1D<Scalar: TensorFlowFloatingPoint>: Layer {
     /// (input width + 2 * padding size - (dilation * (filter width - 1) + 1)) / stride + 1
     ///
     /// and padding size is determined by the padding scheme.
-    /// - Note: Padding size equals zero when using `.valid`.
     ///
     /// - Parameter input: The input to the layer [batch count, input width, input channel count].
     /// - Returns: The output of shape [batch count, output width, output channel count].
+    ///
```

Comment:
Remove empty line.

Reply:
Oops sorry about that. The other occurrences of `- Note:` in the file have an empty line before them.

Reply:
No worries! It would be great to tidy up all of other occurrences as well.

Reply:
I'm on it!